### DISCUSSION TITLE
API Key error

### DISCUSSION THREAD

**Discussion by IuriiDolganov** [assafelovic/gpt-researcher#53] [Q&A]
Hi! 
I wanted to try gpt-researcher, but I keep getting the following error:

openai.error.AuthenticationError: Incorrect API key provided: {sk-TcQG*****************************************ZRo}. You can find your API key at https://platform.openai.com/account/api-keys.

I have checked and regenerated API keys multiple times, but it does not fix the issue.
Does gpt-researcher require a chatGPT+ subscription to work?

Iurii

**Comment by KakashiJiyayou** [comment]
I have the same issue, did you able to fix it ??

**Reply by cyb3run1c0rn** [reply]
You do not need a chat GPT plus subscription, but you will need an account with an API key set up for billing.

Make sure to add your API string correctly to the .env file.

From a command prompt run 'nano .env'

Add the below text, replacing "sk_insert_API_KEY_HERE" with your api key, there should be no spaces at the end.

OPENAI_API_KEY=sk_insert_API_KEY_HERE

**Reply by joma1230** [reply]
wait pls)
